system/include/wasi/api.h

```diff
@@ -29,6 +29,9 @@
 #include <stddef.h>
 #include <stdint.h>
 
+#undef _Static_assert
+#define _Static_assert(X, Y)
+
```

Comment:
Nice!  :)

Reply:
This reminds me of the old `#define private public`

Reply:
Maybe just wrap the failing asserts in `#ifndef __wasm64__`?  It would be a little more self describing and less intrusive. 

Or at least don't disable the asserts on wasm32... that just seems rude.